Tasks/InstallSSHKey/installsshkey-util.ts

```diff
@@ -24,18 +24,9 @@ function execSshAddPassphraseSync(tool, args, passphrase):Q.Promise<boolean> {
     var defer = Q.defer<boolean>();
     let success = true;
 
-    let options:trm.IExecOptions = <trm.IExecOptions> {
-        cwd: process.cwd(),
-        env: process.env,
-        silent: false,
-        failOnStdErr: false,
-        ignoreReturnCode: true,
-        windowsVerbatimArguments: false,
-        outStream:<stream.Writable>process.stdout,
-        errStream:<stream.Writable>process.stderr
-    };
-
-    let cp = child.spawn(tool, args, {});
+    let cp = child.spawn(tool, args, {
```

Comment:
Go with what you have since you have verified it and considered the behavior already.